src/FNAPlatform/SDL2_FNAPlatform.cs

```diff
@@ -990,6 +990,56 @@ public static void UnregisterGame(Game game)
 			}
 		}
 
+		public static bool NeedsPlatformMainLoop()
+		{
+			return SDL.SDL_GetPlatform().Equals("Emscripten");
+		}
+
+		public static void RunPlatformMainLoop(Game game)
+		{
+			if (SDL.SDL_GetPlatform().Equals("Emscripten"))
+			{
+				emscriptenGame = game;
+				emscripten_set_main_loop(
+					RunEmscriptenMainLoop,
+					0,
+					1
+				);
+			}
+		}
+
+		#endregion
+
+		#region Emscripten Main Loop
+
+		// FIXME: Where do these belong?
```

Comment:
This seems fine to me - if someone honestly wants to try an SDL-less Emscripten platform this will be the least of their problems.